### PR TITLE
Added rule for Base64 decoding per 9efa86b43b4367bcdc1591aee59bda25

### PR DESCRIPTION
**Pull request by drfuzzer**
Added a rule to detect Base64-decoding in an unusual case where the table uses 256 dwords, and was not covered by previous rule.

### CODE REVIEW

File: data-manipulation/encoding/base64/decode-data-using-base64.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - or:
+        - mnemonic: sar
+        - mnemonic: shr
+      - match: contain loop
+      - number: 2
+      - number: 3
+      - number: 4
+      - number: 6
+      - number: 0xF
+      - bytes: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF 3E 00 00 00 FF FF FF FF FF FF FF FF FF FF FF FF 3F 00 00 00 34 00 00 00 35 00 00 00 36 00 00 00 37 00 00 00 38 00 00 00 39 00 00 00 3A 00 00 00 3B 00 00 00 3C 00 00 00 3D 00 00 00 FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
```
Comment (williballenthin) [inline data-manipulation/encoding/base64/decode-data-using-base64.yml:26]:
```suggestion
      - bytes: FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF 3E 00 00 00 FF FF FF FF FF FF FF FF FF FF FF FF 3F 00 00 00 34 00 00 00 35 00 00 00 36 00 00 00 37 00 00 00 38 00 00 00 39 00 00 00 3A 00 00 00 3B 00 00 00 3C 00 00 00 3D 00 00 00 FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF = hardcoded base64 translation table (first 64 of 256 dwords)
```

File: data-manipulation/encoding/base64/decode-data-using-base64.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: decode data using Base64
```
Comment (williballenthin) [inline data-manipulation/encoding/base64/decode-data-using-base64.yml:3]:
```suggestion
    name: decode data using Base64 via dword translation table
```

updating name to differentiate against the other base64 rule we have (https://github.com/fireeye/capa-rules/blob/master/data-manipulation/encoding/base64/encode-data-using-base64.yml)

Comment (williballenthin) [reply data-manipulation/encoding/base64/decode-data-using-base64.yml:3]: will need to change filename to reflect this new name, too

**Review by williballenthin — CHANGES_REQUESTED** [review]
see inline suggestions, otherwise, looking good!

**Comment by williballenthin** [comment]
can you submit a PR for the testfile, too? https://github.com/fireeye/capa-testfiles/

**Comment by mr-tz** [comment]
great, thanks!

**Pull request merged**